lib/shoryuken/options.rb

```diff
@@ -148,7 +148,7 @@ def on(event, &block)
     end
 
     def server?
-      defined?(Shoryuken::CLI)
+      defined?(Shoryuken::CLI) || inline_executor?
```

Comment:
Allow yielding with InlineExecutor
https://github.com/phstc/shoryuken/blob/be59f61df31565c63b0840a4655d03ab849fcc81/lib/shoryuken/options.rb#L96-L98

Should I rewrite below??

```rb
def configure_server
  yield self if server? || inline_executor?
end
```